src/__tests__/object.test.ts

```diff
@@ -60,3 +60,16 @@ test('nonstrict', () => {
       unknown: 'asdf',
     });
 });
+
+test('object error', () => {
+  try {
+    z.object({ points: z.number().error({e:1}), dots: z.boolean().error({e:2}) })
+    .parse({
+      points: '1',
+      dots: 1,
+    });
+  } catch (e) {
+    expect(e.errors[0].details).toEqual({e:1});
```

Comment:
Should we add tests for the `error.path`?

 ```
 expect(e.errors[0].path).toEqual('points');
````

Reply:
As I understand correctly, `e.errors[0].message` always equals to `''`?

Reply:
I have added ability to override message or use default. test updated